README.md

```diff
@@ -60,7 +60,7 @@ nix run github:schwarzit/go-template
 
 `go/template`'s `gt` CLI requires at least the following executables on `$PATH` to run succesfully:
 
-- Go >= 1.15
+- Go >= 1.20
```

Comment:
```suggestion
- Go >= 1.21
```

The go mod is at 1.21 in this PR, so it's required to have Go 1.21 to build the binaries.

Reply:
Done and thanks! :D I did it directly when Ctrl+F for occurrences, sorry I didn't take your suggestion here :)

Reply:
No need to be sorry, I only use the suggestions because they look cool. I would also rather commit the changes in the IDE. :)